community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/setup.py

```diff
@@ -597,7 +597,13 @@ def setup_cloud_ops() -> None:
     with open("/etc/google-cloud-ops-agent/config.yaml", "w") as f:
         yaml.safe_dump(file, f, sort_keys=False)
 
-    run("systemctl restart google-cloud-ops-agent.service", timeout=90)
+    try:
+        run("systemctl restart google-cloud-ops-agent.service", timeout=30)
+    except subprocess.TimeoutExpired as e:
+        result=run("journalctl -u google-cloud-ops-agent-opentelemetry-collector.service", timeout=30, shell=True) #Gives more in-depth logs than cloud ops agent
+        if result.stdout:
+            log.error(f"google-cloud-ops-agent.service did not restart within 30s. System logs for google-cloud-ops-agent-opentelemetry-collector.service:\n{result.stdout}")
+        raise e
```

Comment:
* instead of `raise e` do `raise` to not mess up stacktrace;
* Do `log.error` unconditionally ?